OpenOversight/tests/routes/test_officer_and_department.py

```diff
@@ -1674,18 +1674,24 @@ def test_browse_filtering_allows_good(client, mockdata, session):
         )
 
         filter_list = rv.data.decode(ENCODING_UTF_8).split("<dt>Rank</dt>")[1:]
-        assert any(
-            "<dd>{}</dd>".format(job.job_title) in token for token in filter_list
-        )
+        assert any("{}".format(job.job_title) in token for token in filter_list)
+        assert any("<dd>" in token for token in filter_list)
+        assert any("</dd>" in token for token in filter_list)
 
         filter_list = rv.data.decode(ENCODING_UTF_8).split("<dt>Unit</dt>")[1:]
-        assert any("<dd>{}</dd>".format(unit.descrip) in token for token in filter_list)
+        assert any("{}".format(unit.descrip) in token for token in filter_list)
+        assert any("<dd>" in token for token in filter_list)
+        assert any("</dd>" in token for token in filter_list)
```

Comment:
The changes to these assertions maybe feel like we aren't testing what we previously were (since order no longer matters with the new tests). Is it possible to change the assertions so they test the same thing even if there is whitespace in between?

Reply:
I'll give it a shot, yup! It's replicated in the next PR (the image filtering one), so I'll make sure to make whatever change I make on this PR on that one as well.

Reply:
Awesome, thank you! I wonder if we could do something like `combined = "".join([token.strip() for token in filter_list])`, then maybe the assertion can remain the same?

Reply:
That sounds much simpler than the regex solution I was thinking of trying. I'll give it a shot tomorrow and respond back here.

Reply:
I tried your solution and my overly simple solution of `filter_list = [token.replace(" ", "").replace("\n", "") for token in filter_list` to no avail.

The problem was that between the `<dd>` elements you had a starting line break and A LOT of spaces. To address that, I created this function and created all of the `filter_list`s with it.
```python
def normalize_tokens_for_comparison(html_str: str, split_str: str):
  """Remove new lines, leading, and closing spaces between <dd> elements in
  formatted HTML."""
  parsed_list = html_str.data.decode(ENCODING_UTF_8).split(split_str)[1:]
  parsed_list = [re.sub(r"<dd>\n\s+", "<dd>", token) for token in parsed_list]
  parsed_list = [re.sub(r"\n\s+</dd>", "</dd>", token) for token in parsed_list]
  return parsed_list
```

I've since added it to both branches.